src/app/address/steps.js

```diff
@@ -0,0 +1,14 @@
+const address = require("./controllers/address");
+
+module.exports = {
+  "/": {
+    resetJourney: true,
+    entryPoint: true,
+    skip: true,
+    next: "address",
+  },
+  "/address": {
+    controller: address,
+    next: "/",
```

Comment:
Does this create an infinite loop of redirects?

Reply:
For now yep it will redirect back to the start of the form yep, we should catch up on what the next steps should be as this is added temporary.